### PR TITLE
Fixes #1160: Moved some high frequency info level logs to debug.

### LINKED ISSUE #1160: Too many link creation/deletion INFO log messages flooding the router log
There is one link creation/deletion INFO level log message pair that show up on every creation and teardown of every TCP connection in the TCP adaptor. 

At a high TCP connection rate, these log message can flood the log which take up a lot of storage and these logs are not really all that useful.

Go thru the INFO level logs and clean up the logs. Only two types of messages should show at the INFO level - 
1. One time messages like during router startup or adaptor startup etc.
2. Configuration changes made to the router

### PR DESCRIPTION
**Pull request by ganeshmurthy**
Added some new info level logs that show up on router schema changes.

### CODE REVIEW

File: src/vanflow.c [hunk 1]
```diff
@@ -770,7 +770,8 @@ static void _vflow_emit_record_as_log_TH(vflow_record_t *record)
     }
 
     record->never_logged = false;
-    qd_log(LOG_FLOW_LOG, QD_LOG_INFO, "%s", line);
+    // This log statement emits a single record as a log event
+    qd_log(LOG_FLOW_LOG, QD_LOG_DEBUG, "%s", line);
```
Comment (ted-ross) [inline src/vanflow.c:774]: I think this is the one you must not change.  This removes ALL flow-related content from the logs.

**Review by ted-ross — CHANGES_REQUESTED** [review]
Looks good overall.  I have one comment that we should discuss before you merge.

File: src/router_core/connections.c [hunk 2]
```diff
@@ -136,7 +136,8 @@ qdr_connection_t *qdr_connection_opened(qdr_core_t                   *core,
 
     pn_data_format(connection_info->connection_properties, props_str, &props_len);
 
-    qd_log(LOG_ROUTER_CORE, QD_LOG_INFO,
+    // High frequency log message, setting to QD_LOG_DEBUG
```
[the remark this answers is not in the document]
Comment (kgiusti) [reply src/router_core/connections.c:139]: Also maybe add the "role=<role>" to the log?

File: src/router_core/connections.c [hunk 3]
```diff
@@ -1821,7 +1826,8 @@ static void qdr_connection_closed_CT(qdr_core_t *core, qdr_action_t *action, boo
 
     qdrc_event_conn_raise(core, QDRC_EVENT_CONN_CLOSED, conn);
 
-    qd_log(LOG_ROUTER_CORE, QD_LOG_INFO, "[C%" PRIu64 "] Connection Closed", conn->identity);
+    // High frequency log message, setting to QD_LOG_DEBUG level.
+    qd_log(LOG_ROUTER_CORE, QD_LOG_DEBUG, "[C%" PRIu64 "] Connection Closed", conn->identity);
```
Comment (kgiusti) [inline src/router_core/connections.c:1830]: Ditto INFO if backbone conn?

**Review by kgiusti — APPROVED** [review]
Some inline nits - only suggestions for you to consider.

File: src/router_core/connections.c (hunk 2 above)
[the remark this answers is not in the document]
Comment (ganeshmurthy) [reply src/router_core/connections.c]: I have made the requested changes. Please take a look.

**Pull request merged**